app/assets/images/payment_icons/netbanking.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 24"><title>bank</title><rect x="0.5" y="0.5" width="37" height="23" rx="3" ry="3" style="fill:#fff;stroke:#000;stroke-opacity:0.07000000029802322"/><polygon points="19 4.5 11.5 10.13 26.5 10.13 19 4.5"/><path d="M25.56,17.63H12.44a0.94,0.94,0,0,0-.94.94V19.5h15V18.56A0.94,0.94,0,0,0,25.56,17.63Z"/><rect x="19.94" y="11.06" width="1.88" height="5.63"/><rect x="23.69" y="11.06" width="1.88" height="5.63"/><rect x="16.19" y="11.06" width="1.88" height="5.63"/><rect x="12.44" y="11.06" width="1.88" height="5.63"/></svg>
```

Comment:
Is it possible to improve the resolution of the image at all? The bottom part looks a bit out of focus.